server/index.html

```diff
@@ -0,0 +1,105 @@
+    </div>
+    <div class="container" style="display: flex; justify-content: center;">
+    <div class="game-grid" style="display: grid; grid-template-columns: 60px 60px 60px; grid-template-rows: 60px 60px 60px;">
+      <div class="game-grid-item" id="grid-1" style="border: 1px solid black; height: 50px; width: 50px;">1</div>
+      <div class="game-grid-item" id="grid-2" style="border: 1px solid black; height: 50px; width: 50px;">2</div>
+      <div class="game-grid-item" id="grid-3" style="border: 1px solid black; height: 50px; width: 50px;">3</div>
+      <div class="game-grid-item" id="grid-4" style="border: 1px solid black; height: 50px; width: 50px;">4</div>
+      <div class="game-grid-item" id="grid-5" style="border: 1px solid black; height: 50px; width: 50px;">5</div>
+      <div class="game-grid-item" id="grid-6" style="border: 1px solid black; height: 50px; width: 50px;">6</div>
+      <div class="game-grid-item" id="grid-7" style="border: 1px solid black; height: 50px; width: 50px;">7</div>
+      <div class="game-grid-item" id="grid-8" style="border: 1px solid black; height: 50px; width: 50px;">8</div>
+      <div class="game-grid-item" id="grid-9" style="border: 1px solid black; height: 50px; width: 50px;">9</div>
+    </div>
+  </div>
+    <div class="game-log">
+      <p class="game-log-text" id="logs" style="display: block">Logs go here</p>
+    </div>
+  </div>
+
+  <script>
+    const ws = new WebSocket("ws://localhost:8080");
+
+    ws.addEventListener("open", () => {
+      console.log("We are connected!");
+
+      document.getElementById("input").addEventListener("keyup", event => {
+        if (event.key === "Enter") {
+          let inputVal = event.target.value;
+          let inputObj = {
+            id: "input",
+            input: inputVal,
+          }
+          ws.send(JSON.stringify(inputObj))
+          event.target.value = "";
+        }
+      })
+
+      const gridArray = document.getElementsByClassName("game-grid-item")
+      for(let i = 0; i < gridArray.length; i++) {
+        gridArray[i].addEventListener("click", event => {
+          const id = gridArray[i].getAttribute("id").split("-")[1]
+          let idObj = {
+            grid: Number(id),
+            id: "grid"
+          }
+          ws.send(JSON.stringify(idObj));
+          gridArray[i].innerHTML = document.getElementById("current-team").innerHTML.split(" ")[1];
+          gridArray[i].setAttribute("style", "pointer-events:none")
+        })
+
+      }
+
+      ws.addEventListener("message", (message) => {
+        let msg = JSON.parse(message.data)
+        console.log(msg)
+
+        const logElement = document.getElementById("logs")
+        let currentText = logElement.innerHTML;
+        switch(msg.id) {
+          case "prompt":
+            document.getElementById("prompt").innerHTML = msg.prompt;
+            document.getElementById("logs").innerHTML = msg.prompt + "<br>" + currentText;
```

Comment:
(not now, but...)

* frontend state object
* switch statement becomes updateState(state, msg) -> newState
* separate function updateDom(state)